src/OrchardCore.Modules/OrchardCore.Tenants/TenantsOptions.cs

```diff
@@ -0,0 +1,6 @@
+namespace OrchardCore.Tenants;
+
+public class TenantsOptions
+{
+    public bool TenantRemovalAllowed { get; set; }
```

Comment:
While the name is `TenantsOptions`, we could rename the property to `AllowRemoval`

Reply:
In the `appsettings.json` sample we use the same `TenantRemovalAllowed` field name.

The `OrchardCore_Tenants` section may be used to add options tied to other features of `OC.Tenants` so just to be clear that it is related to the `TenantRemoval` functionality (which is not a module feature).